src/elements/content-sidebar/activity-feed/comment/__tests__/Replies.test.js

```diff
@@ -170,6 +172,22 @@ describe('elements/content-sidebar/ActivityFeed/comment/Replies', () => {
         expect(replyCreate).toBeCalledWith('Batman');
     });
 
+    test('should call onReplyDelete when reply is deleted', () => {
+        getWrapper({ replies: [comment] });
```

Comment:
Could we add another reply and make sure that the chosen reply is the correct reply that is deleted?

Reply:
I think line 188 should account for that. But it would be nice to add a few comments with different ids and just select a comment from the list in the replies component, and verify that reply delete was called with the right id..

Reply:
Yea I can try that. I was running into issues with the test ids being duplicates whenever multiple comments were added but can find a way to work around that.

Reply:
@JChan106 does the current dropdown logic account for proper users? This change only makes the dropdown button work. If not, I'll update the dropdown logic to include.

Reply:
I updated my test to include multiple comments and verify that the expected comment is clicked. 

I began writing a test to ensure the delete option shows/hides based on the user (ie permissions) but found a much more comprehensive test for this already exists https://github.com/box/box-ui-elements/blob/master/src/elements/content-sidebar/activity-feed/comment/__tests__/BaseComment.test.js#L116